tests/trx_generator/main.cpp

```diff
@@ -112,6 +119,22 @@ int main(int argc, char** argv) {
             return INITIALIZE_FAIL;
          }
       }
+
+      if(vmap.count("spinup-time-us")) {
+         if(spinup_time_us < 0) {
+            ilog("Initialization error: spinup-time-us cannot be negative");
+            cli.print(std::cerr);
+            return INITIALIZE_FAIL;
+         }
+      }
+
```

Comment:
Should a check be added for "monitor-max-lag-percent" being in a reasonable % range?  At the very least check it is in percentage range 0-100?

Reply:
added validation